servers/rendering/renderer_rd/environment/sky.cpp

```diff
@@ -418,6 +425,8 @@ void SkyRD::ReflectionData::create_reflection_fast_filter(bool p_use_arrays) {
 			}
 		}
 		RD::get_singleton()->draw_command_end_label(); // Filter radiance
+
+		copy_effects->calculate_sh_from_cubemap(downsampled_radiance_cubemap, sh_coeff_buffer);
```

Comment:
Using the `downsampled_radiance_cubemap` might partly explain why the SH version is missing highlights. It uses either a half res cubemap (when using the "high quality" path) or a 64x64 cubemap (when using the "fast filter" path). In either case its a lower res cubemap that is already blurred.

Reply:
I was a bit confused on _where_ I should grab the texture from, I thought I got it from the right place (resolution is 128x128 IIRC) but it's definitely possible that it's wrong.

Reply:
Good catch that perhaps the problem is simply the wrong input.